benchmark/camelyon/pure_substrafl/register_assets.py

```diff
@@ -42,7 +42,7 @@ def instantiate_clients(
     mode: substra.BackendType = substra.BackendType.LOCAL_SUBPROCESS,
     n_centers: Optional[int] = 2,
     conf: Optional[dict] = None,
```

Comment:
Looks like it's a `Optional[list[dict]]` (same in docstring)

Reply:
Indeed